nuxeo-webdav/src/main/java/org/nuxeo/ecm/webdav/backend/SimpleBackend.java

```diff
@@ -358,20 +335,20 @@ public DocumentModel moveItem(DocumentModel source, PathRef targetParentRef) {
     }
 
     @Override
-    public DocumentModel moveItem(DocumentModel source, DocumentRef targetParentRef, String name)
-            {
+    public DocumentModel moveItem(DocumentModel source, DocumentRef targetParentRef, String name) {
         cleanTrashPath(targetParentRef, name);
-        DocumentModel model = getSession().move(source.getRef(), targetParentRef, name);
-        getPathCache().put(parseLocation(targetParentRef.toString()) + "/" + name, model);
-        getPathCache().remove(source.getPathAsString());
-        return model;
+        BlobHolder blobHolder = source.getAdapter(BlobHolder.class);
+        Blob blob = blobHolder.getBlob();
+        blob.setMimeType(null);
```

Comment:
Please add a comment explaining why we set the MIME type to null, it's not immediately intuitive. Actually I'm not sure it's needed, if the content doesn't change the MIME type shouldn't need to change (even if the extension is different).

Reply:
I've copied this from the method above.
```
// as the name may have changed, reset the mime type so that the correct one will be computed
```
Should i remove this line still ?

Reply:
Yes let's keep the MIME type unchanged. It's what the previous code did and it seemed to work ok. In the future if this causes problem we'll fix this as a separate improvement.